README.md

```diff
@@ -1,2 +1,67 @@
+  * [Algebird](https://github.com/twitter/algebird)
+  * [Bijection](https://github.com/twitter/bijection)
+  * [Chill](https://github.com/twitter/chill)
+  * [Storehaus](https://github.com/twitter/storehaus)
+  * [Tormenta](https://github.com/twitter/tormenta)
+
+## Contributing and Committership
+
+It is not necessary to become a committer in order to contribute to the Analytics Infrastructure OSS Ecosystem. Anyone is welcome to contribute to Twitter's OSS projects, and pull requests and bug reports are greatly appreciated.
+We ask that contributors follow [Twitter's Open Source Code of Conduct](https://engineering.twitter.com/opensource/code-of-conduct). Additionally, see the [Typelevel Code of Conduct](http://typelevel.org/conduct) for specific examples of harassing behavior that are not tolerated.
+
+In addition to contributing, anyone can become a committer. The role of committer is not purely a technical one, it is also a project stewardship and maintenance role. Of course, anyone is welcome to contribute to the project without becoming a committer - you only need to be a committer if you want to play a leadership or stewardship role within the project. Only committers can merge pull requests, because only committers have write access to the repositories.
+
+A prospective committer should demonstrate that they have:
+
+  * Sustained non-trivial, well-tested contributions accepted into the project
+  * Community involvement, including reviewing other's contributions (code reviews), participating in design discussions, attending project sync-ups, and contributing to project maintenance (documentation, tech-debt cleanup)
+
+In order to add a new committer to a project, the existing active committers in that project must vote, using the same voting process as the one for releases described below, with at least one -0 or higher vote from a Twitter employee committer.
+
+Each project contains a COMMITTERS.md file at the root of the project that lists the current and emeritus committers. If a committer hasn’t contributed to the project (i.e. by participating in reviews, or by submitting contributions) for a period of ONE year, their status will be moved to Emeritus, and they will no longer have commit/vote/release privileges.
+
+## Code Review and Merge Process
+
+Committers (and other contributors) will review your contribution (i.e. a pull request). Each contribution must get at least one +1 from an active committer (who is not yourself) before merging. If the initial reviewer is unsure whether to +1 something or not, they should involve other committers for testing/validation of the contribution. No PRs should be merged if there are open issues that have not been discussed, or if there are any -1s without further discussion, or if the build is not green. Any committer can merge the change after the review process is complete and the build is green.
```

Comment:
Yeah I guess I was getting at the scenario where the committer might not explicitly mention one of the other committers to review. Might be a good idea to mandate waiting a day or two (unless its an urgent fix) to ensure in case anyone else has concerns they can voice them before it gets merged.